### ISSUE TITLE
Missing requirement: requests

### ISSUE THREAD

**Issue by raveydaveygravy** [smicallef/spiderfoot#125]
I saw the error below when running sf.py on Ubuntu.

```
root@ubuntu:~/spiderfoot# ./sf.py 0.0.0.0:5001
Traceback (most recent call last):
  File "./sf.py", line 54, in <module>
    from sflib import SpiderFoot
  File "/root/spiderfoot/sflib.py", line 24, in <module>
    import requests
ImportError: No module named requests
```

sf.py ran successfully after installing requests. Is this dependency missing from the documentation? Thanks for this awesome project by the way

```
root@ubuntu:~/spiderfoot# pip install requests
Collecting requests
  Downloading requests-2.13.0-py2.py3-none-any.whl (584kB)
    100% |████████████████████████████████| 593kB 870kB/s
Installing collected packages: requests
Successfully installed requests-2.13.0
root@ubuntu:~/spiderfoot# ./sf.py 0.0.0.0:5001
Starting web server at http://0.0.0.0:5001 ...

```

**Comment by smicallef** [comment]
Thanks.. actually I thought requests came bundled by default with most distros. What version of Ubuntu are you using?

**Comment by roxy9990** [comment]
16.04

**Comment by smicallef** [comment]
OK, I am going to add requests as a (documented) requirement in 2.9. For now I would say to run apt-get install python-requests.

**Comment by smicallef** [comment]
Should now be addressed in 2.9.0.
